api/pubsub.py

```diff
@@ -27,7 +29,8 @@ async def create(self, *args, **kwargs):
         await pubsub._init_sub_id()
         return pubsub
 
-    def __init__(self, host='redis', db=1):
+    def __init__(self, host=Settings().redis_host,
+                 db=Settings().redis_db_number):
```

Comment:
There is already a `Settings` object created in `self._settings`. So I think the default values in `__init__` should be `None` and then values from `self._settings` should be used.  For example:
```python
if host is None:
    host = self._settings.host
```